cms/djangoapps/contentstore/views/course.py

```diff
@@ -412,12 +412,16 @@ def _accessible_courses_list_from_groups(request):
     """
     List all courses available to the logged in user by reversing access group names
     """
+    def filter_ccx(course_access):
+        """ CCXs cannot be edited in Studio and should not be shown in this dashboard """
+        return not isinstance(course_access.course_id, CCXLocator)
```

Comment:
@aamir-khan Added this to check on course_id instance. We can use either way :). Addressing your recommendation.